data/Language/Rust.yaml

```diff
@@ -0,0 +1,79 @@
+identifier: Rust
+full_name: Rust
+wikidata: Q575650
+aliases:
+  - Rust
+  - rs
+extensions:
+  - rs
+  - rlib
+line_continuation: '\'
+delimiters:
+  # comment delimiters
+  - double_slash
+  - triple_slash
+  - doc_comment
+  - multiline_slash_star
+  # string delimiters:
+  - c_char
+  - double_quote_slash_escape
+  - curly_braces
+datatypes:
+  - boolean_rust
+keywords:
+ - as
```

Comment:
wrong indentation: expected 2 but found 1 (indentation)

*Origin: YAMLLintBear, Section: `yaml`.*

Reply:
The bot to the rescue.  I'll fix the indentation